vcfa/sample_vcfa_test_config.json

```diff
@@ -0,0 +1,49 @@
+{
+  "//": "Sample test configuration file for VCFA",
+  "//": "Rename this file to vcfa_test_config.json and change the values to suit your system",
+  "//": "You can safely remove the comments or leave them. The parser will ignore them.",
+  "provider": {
+    "//"  : "This section contains credentials related to the VCFA connection of Sys or Org user",
+    "user": "root",
```

Comment:
Shouldn't this be `serviceadministrator`?

Reply:
True, fixed